src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -1209,6 +1218,29 @@ public OpenLoadFlowParameters setFictitiousGeneratorVoltageControlCheckMode(Fict
         return this;
     }
 
+    public boolean isAreaInterchangeControl() {
+        return areaInterchangeControl;
+    }
+
+    public OpenLoadFlowParameters setAreaInterchangeControl(boolean areaInterchangeControl) {
+        this.areaInterchangeControl = areaInterchangeControl;
+        return this;
+    }
+
+    public String getAreaInterchangeControlAreaType() {
+        return areaInterchangeControlAreaType;
+    }
+
+    public OpenLoadFlowParameters setAreaInterchangeControlAreaType(String areaInterchangeControlAreaType) {
+        this.areaInterchangeControlAreaType = Objects.requireNonNull(areaInterchangeControlAreaType);
+        return this;
+    }
+
+    public OpenLoadFlowParameters setVoltagePerReactivePowerControl(boolean voltagePerReactivePowerControl) {
```

Comment:
@vmouradian it looks like you moved this function by error

Reply:
:sweat_smile: oups. Changed in 5a29ee99ce070c764b03988fb43c2636eb909798

Reply:
I think there is still a problem: you didn't move it back to its initial place. See the diff view to check if you rolled-back correctly all those modifications
![image](https://github.com/user-attachments/assets/606c081e-e42f-4872-afea-b4771a3a694a)

Reply:
Do not remember the last commit on this but the diff seems ok now